Tests/ApolloTests/Cache/SQLite/CachePersistenceTests.swift

```diff
@@ -79,67 +79,68 @@ class CachePersistenceTests: XCTestCase {
       self.waitForExpectations(timeout: 2, handler: nil)
     }
   }
-
-  func testFetchAndPersistWithPeriodArguments() throws {
-    let query = SearchQuery(term: "Luke.Skywalker")
-    let sqliteFileURL = SQLiteTestCacheProvider.temporarySQLiteFileURL()
-
-    try SQLiteTestCacheProvider.withCache(fileURL: sqliteFileURL) { (cache) in
-      let store = ApolloStore(cache: cache)
-
-      let server = MockGraphQLServer()
-      let networkTransport = MockNetworkTransport(server: server, store: store)
-
-      let client = ApolloClient(networkTransport: networkTransport, store: store)
-
-      _ = server.expect(SearchQuery.self) { request in
-        [
-          "data": [
-            "search": [
-              [
-                "id": "1000",
-                "name": "Luke Skywalker",
-                "__typename": "Human"
-              ]
-            ]
-          ]
-        ]
-      }
-      let networkExpectation = self.expectation(description: "Fetching query from network")
-      let newCacheExpectation = self.expectation(description: "Fetch query from new cache")
-
-      client.fetch(query: query, cachePolicy: .fetchIgnoringCacheData) { outerResult in
-        defer { networkExpectation.fulfill() }
-
-        switch outerResult {
-        case .failure(let error):
-          XCTFail("Unexpected error: \(error)")
-          return
-        case .success(let graphQLResult):
-          XCTAssertEqual(graphQLResult.data?.search?.first??.asHuman?.name, "Luke Skywalker")
-          // Do another fetch from cache to ensure that data is cached before creating new cache
-          client.fetch(query: query, cachePolicy: .returnCacheDataDontFetch) { innerResult in
-            try! SQLiteTestCacheProvider.withCache(fileURL: sqliteFileURL) { cache in
-              let newStore = ApolloStore(cache: cache)
-              let newClient = ApolloClient(networkTransport: networkTransport, store: newStore)
-              newClient.fetch(query: query, cachePolicy: .returnCacheDataDontFetch) { newClientResult in
-                defer { newCacheExpectation.fulfill() }
-                switch newClientResult {
-                case .success(let newClientGraphQLResult):
-                  XCTAssertEqual(newClientGraphQLResult.data?.search?.first??.asHuman?.name, "Luke Skywalker")
-                case .failure(let error):
-                  XCTFail("Unexpected error with new client: \(error)")
-                }
-                _ = newClient // Workaround for a bug - ensure that newClient is retained until this block is run
-              }
-            }
-          }
-        }
-      }
-
-      self.waitForExpectations(timeout: 2, handler: nil)
-    }
-  }
+  
+  #warning("TODO")
```

Comment:
Was this meant to make it into the PR and fixed in subsequent PRs? If so we'll need to ensure it's removed before the next alpha release.

Reply:
It's the only test that isn't BUILDING in Apollo tests right now, because it references a query generated from StarWarsAPI. Don't think it's crucial for the next alpha, but before Beta for sure! 

It's not commented out because it's failing and there is broken behavior, it just can't compile without the StarWarsAPI, which will be generated once we finish include/skip and generate the `__typename`s.